### PR TITLE
Ft log in page #160911795

### PR DESCRIPTION
**Pull request by akpante3**
#### What does this PR do?
it logs in  an already signed up user into the app

#### Description of Task to be completed?
1. wrote authentications for the login route
2. made use of fetch API to consume the sign-up API endpoint

#### How should this be manually tested?
on your console
- cd server
- npm start

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?

[#160911795](https://www.pivotaltracker.com/story/show/160911795)

#### Screenshot
N/A
#### Questions:
N/A

### CODE REVIEW

File: UI/newfood.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <br>
+            <footer>
+                <h3>fast-food-fast   &copy 2018</h3>
+            
+            </footer> 
+    </article>
+    <script src="./js/jquery.min.js"></script>
+    <script src="./js/auth.js"></script> 
+</body>
+</html>
```
Comment (KaiserPhemi) [inline UI/newfood.html]: #### Nitpick
Insert a new line at the end of file.

Comment (KaiserPhemi) [reply UI/newfood.html]: Please do this for all other files as well. I do not need to comment on every file without a new line at the end.

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
#### Critical
Document your functions/method using `jsdoc`. Also attend to all feedback

File: UI/newfood.html (hunk 1 above)
Comment (akpante3) [reply UI/newfood.html]: okay

**Pull request merged**